src/main/java/ua/lviv/iot/parkingServer/datastorage/VehicleFileStore.kt

```diff
@@ -0,0 +1,20 @@
+package ua.lviv.iot.parkingServer.datastorage
+
+import org.springframework.stereotype.Component
+import ua.lviv.iot.parkingServer.model.Vehicle
+import ua.lviv.iot.parkingServer.model.enums.VehicleType
+
+@Component
+class VehicleFileStore : AbstractFIleStore<Vehicle>() {
+    override fun getRecordName(): String = "vehicle"
+
+    override fun convert(values: List<String>): Vehicle {
+        return Vehicle(
+            values[0].toLong(),
+            values[1],
+            VehicleType.valueOf(values[2]),
+            values[3].toDouble(),
+            values[4].toBoolean()
```

Comment:
The code will be much more readable if you use named parameters here

https://www.baeldung.com/kotlin/default-named-arguments

Reply:
✅